backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/PostgresFieldOfLawRepositoryImpl.java

```diff
@@ -72,119 +76,57 @@ public Slice<FieldOfLaw> findAllByOrderByIdentifierAsc(Pageable pageable) {
 
   @Override
   @Transactional
-  public List<FieldOfLaw> findBySearchTerms(String[] searchTerms) {
-    if (searchTerms == null || searchTerms.length == 0) {
-      return Collections.emptyList();
+  public List<FieldOfLaw> findByCombinedCriteria(
+      String identifier, String[] descriptionSearchTerms, String[] normSearchTerms) {
+    CriteriaBuilder cb = entityManager.getCriteriaBuilder();
+    CriteriaQuery<FieldOfLawDTO> cq = cb.createQuery(FieldOfLawDTO.class);
+    Root<FieldOfLawDTO> fieldOfLawRoot = cq.from(FieldOfLawDTO.class);
+    fieldOfLawRoot.fetch("norms", JoinType.LEFT);
```

Comment:
question: Do we always need to fetch with `norms` or only when we query by it? (There is also a checkbox "Mit Normen", not sure if that is related.)
But even if we don't always need it it would just be a performance optimization, so no blocker.